src/components/Stepper/Progress/Progress.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import VisuallyHidden from '../../VisuallyHidden';
+
+import * as S from './Progress.style';
+
+export type ProgressProps = React.PropsWithChildren<any> & {
+	orientation: 'horizontal' | 'vertical';
+	valuemin?: number;
```

Comment:
why didn't you used camel case, or simply `min`, `max`, `value` to fit the HTML progress api ?

Reply:
I was using aria default attributes, but it makes sense to use default `<Progress>` attrs 😅